CONTRIBUTING.md

```diff
@@ -65,29 +65,14 @@ You are welcome to contribute code to Fundamental Library for Angular in order t
 
 There are two important things to consider:
 
-1.  You must be aware of the Apache License (which describes contributions) and **agree to the Contributors License Agreement**. This is common practice in all major Open Source projects. If you are an individual contributor, use _[CLA assistant](https://cla-assistant.io/)_. CLA assistant is an open source tool that integrates with GitHub and enables a one-click-experience for accepting the CLA. If you are contributing on behalf of a company, see the [Company Contributors](#company-contributors) section below for details.
+1.  You must be aware of the Apache License (which describes contributions) and **accept the Developer Certificate of Origin**. This is common practice in major Open Source projects. To make this process as simple as possible, we are using *[CLA assistant](https://cla-assistant.io/)* for individual contributions. CLA assistant is an open source tool that integrates with GitHub very well and enables a one-click experience for accepting the CLA. For company contributors, special rules apply. See the respective section below for details.
```

Comment:
1. You must be familiar with the Apache License (that describes contributions) and **accept the Developer Certificate of Origin**. This is a common practice in major open source projects. To make this process as simple as possible, we are using *[CLA assistant](https://cla-assistant.io/)* for individual contributions. CLA assistant is an open source tool that integrates well with GitHub, providing users with an easy one-click experience when accepting the CLA. For company contributors, special rules apply. See the respective section below for details.